src/NzbDrone.Core/Parser/Parser.cs

```diff
@@ -675,7 +675,7 @@ public static string CleanSeriesTitle(this string title)
             // Replace `%` with `percent` to deal with the 3% case
             title = PercentRegex.Replace(title, "percent");
 
-            return NormalizeRegex.Replace(title).ToLower().RemoveAccent();
+            return ReplaceGermanUmlauts(NormalizeRegex.Replace(title).ToLower()).RemoveAccent();
```

Comment:
Like the `PercentRegex` above these should be separated into their own lines.

Reply:
Is it okay like this?
```
title = NormalizeRegex.Replace(title).ToLower();
return ReplaceGermanUmlauts(title).RemoveAccent();
```

Reply:
I think I'd go one step further and go with:

```
title = NormalizeRegex.Replace(title).ToLower();
title = ReplaceGermanUmlauts(title);

return title.RemoveAccent();
```

Makes it a bit more readable and ideally better shows off the preferred way to implement another similar feature in the future.